src/main/scala/org/broadinstitute/dsde/vault/services/uBAM/IngestServiceHandler.scala

```diff
@@ -81,10 +93,17 @@ case class IngestServiceHandler(requestContext: RequestContext, bossService: Act
   def completeIfDone =
     dmId match {
       case Some(id) =>
-        if (fileCount == bossURLs.size) {
-          log.debug("uBAM ingest complete")
-          requestContext.complete(UBamIngestResponse(id, bossURLs).toJson.prettyPrint)
-          context.stop(self)
+        if (fileCount == bossObjects.size) {
```

Comment:
That will fail because the urls are not populated in the `X-Force-Location` condition. I'll look at another way to try and get at that issue.